### DISCUSSION TITLE
Paired vertical bars do not adjust to the size of the inside expression

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#3136] [Q&A] [answered]
Consider the following code, which produces the expression below. As you can see, the vertical bars do not adjust to the size of the inside fraction. Is there some workaround? Thanks!

```
$
ln |frac(y-2, y)|
$
```

![image](https://github.com/typst/typst/assets/75945439/e17bf89e-9d7a-4c35-8ba2-bd9cfe29ff12)

**Comment by 8LWXpg** [comment, marked as answer]
warp it in [`lr`](https://typst.app/docs/reference/math/lr)

```typst
$
ln lr(|frac(y-2, y)|)
$
```

![image](https://github.com/typst/typst/assets/105704427/e0c7b0e4-51d2-48cf-af44-8f6fb9694f44)

**Reply by psads-git** [reply]
Thanks, @8LWXpg: that works!

**Comment by MDLC01** [comment]
While [`lr`](https://typst.app/docs/reference/math/lr#functions-lr) is a more general solution, in this case you can use [`abs`](https://typst.app/docs/reference/math/lr#functions-abs).

**Reply by psads-git** [reply]
Thanks, @MDLC01: that works too!
